pluggy/parse_funcs.py

```diff
@@ -0,0 +1,66 @@
+import inspect
```

Comment:
I was actually thinking maybe just make this a `utils.py` mod since it's only got the one function and we might later want a place to add miscellaneous stuff.

Reply:
in general - having a utils module/package is a antipattern ^^ - its just a utterly bad name because it caries no meaning